### ISSUE TITLE
Book author information

### ISSUE THREAD

**Issue by Protektor-Desura** [jellyfin/jellyfin-plugin-bookshelf#21]
t would be nice if Jellyfin could pull in from Goodreads or Wikipedia information about the author and display it when you are showing all the books by a specific author along with a picture of the author.

**Comment by spacekookie** [comment]
At the moment it doesn't actually display the author anywhere, right?

**Comment by Protektor-Desura** [comment]
Correct. It doesn't. It just lists all the books by that author is all.

**Comment by hacketiwack** [comment]
If I understood properly, it is not even listing books by authors, but just by the folders that are supposed to be named after authors. And therefore, there is no way to search by author name.
Am I right or did I miss something?

**Comment by jakobwuhrer** [comment]
I'm also not seeing any way to add, import, read, edit, or view author-related metadata, so it seems like there's some sort of bug here

**Comment by bt4ibwem8** [comment]
For me it shows the author, but the problem is Jellyfin is searching metadata about author in TMDB only. So it is not reliable, much better would be if JF search authors in Goodreads or other reliable sources...
